plugins/export.js

```diff
@@ -13,3 +17,4 @@ window.ReactBootstrap = require('react-bootstrap');
 window.PostUtils = {formatText, messageHtmlToComponent};
 window.PropTypes = require('prop-types');
 window.PDFJS = require('pdfjs-dist');
+window.rhsPluginAction = bindActionCreators({showPluginRHS}, store.dispatch);
```

Comment:
Why are we saving an action to the window?
Can't we just let the plugin bind the action?

Reply:
The action definition is in the webapp given that is uses the action name expected by the reducer and I exposed the function ready to just call the action as an export through the window.  

I could move the binding to the plugin and only have action definition in the export if preferred. However, I believe it's an additional step that all plugin clients must do before they call the action.

Reply:
I've removed the binding in the export as suggested. 

Implementing it in the github plugin I see why it does make more sense to have it in the client where they would connect their component to the actions and bind them as usual. 

Also, I've included another change that came up to support multiple plugins registering their RHS component and being able to only show the one that called the action to open it.

Reply:
I think we need to go a step further here. Can we just have the plugin import the action directly instead of passing it though the window? There really isn't a precedent for a plugin importing the webapp and it would be nice to keep the interface. So maybe we could have it returned by `registerRightHandSidebarComponent` instead? I just want to keep the interface clear.

Reply:
Gotcha! Much better now 👍